lib/list-swipe-save.js

```diff
@@ -59,7 +59,7 @@ SwipeSaveList.prototype.init = function(){
     actionThreshold : 120
 	});
 
-  Fliplet.Storage.get('saved_list_' + _this.listSwipe.container.parentNode.dataset.listId, {
```

Comment:
FYI the UUID can also be retrieved using `Fliplet.Widget.getUUID(widgetInstanceId);` (no preference over which method to use)